src/strategies/staked-defi-balance/examples.json

```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "name": "Staked Defi Balance",
+        "strategy": {
+            "name": "staked-defi-balance",
+            "params": {
+                "tokenContractAddress": "0xe685d3CC0be48BD59082eDe30C3b64CbFc0326e2",
+                "symbol": "cFRM",
+                "decimals": 18,
+                "minStakedBalance": "100000000000000000000000",
+                "stakingPoolContractAddress": "0xb4927895cbee88e651e0582893051b3b0f8d7db8",
+                "methodABI": [
+                    {
+                        "inputs": [
+                            {
+                                "internalType": "address",
+                                "name": "id",
+                                "type": "address"
+                            },
+                            {
+                                "internalType": "address",
+                                "name": "staker",
+                                "type": "address"
+                            }
+                        ],
+                        "name": "stakeOf",
+                        "outputs": [
+                            {
+                                "internalType": "uint256",
+                                "name": "",
+                                "type": "uint256"
+                            }
+                        ],
+                        "stateMutability": "view",
+                        "type": "function"
+                    }
```

Comment:
Thanks @taha-abbasi This abi will keep changing from space to space, if not i would recommend you to hard code in strategy itself

Reply:
Hi @ChaituVR we need to keep the ABI as different spaces might use the rewardOf, or stakeOf function, or other functions of the staking contracts. There is a plan in our architecture to use the same strategy and allow multiple use cases based on ABI provided and the space's specific requirements.